operator/hack/olm-operator-install.sh

```diff
@@ -16,34 +16,41 @@ function main() {
   esac
 
   local -r operator_ns="${1:-}"
-  local -r image_tag_base="${2:-}"
+  local -r index_image_repo="${2:-}"
 
   case $# in
   3)
-    local -r index_version="${3:-}"
-    local -r operator_version="${3:-}"
+    local -r index_image_tag="${3:-}"
+    local -r csv_version="${3:-}"
+    local -r operator_channel="latest"
     ;;
   4)
-    local -r index_version="${3:-}"
-    local -r operator_version="${4:-}"
+    local -r index_image_tag="${3:-}"
+    local -r csv_version="${4:-}"
+    local -r operator_channel="latest"
+    ;;
+  5)
+    local -r index_image_tag="${3:-}"
+    local -r csv_version="${4:-}"
+    local -r operator_channel="${5:-}"
     ;;
   *)
-    echo -e "Usage:\n\t$0 [--allow-dirty-tag | -d] <operator_ns> <image_tag_base> <index-version> [<install-version>]" >&2
-    echo -e "Example:\n\t$0 -d index-test quay.io/rhacs-eng/stackrox-operator 3.70.1 3.70.1" >&2
+    echo -e "Usage:\n\t$0 [--allow-dirty-tag | -d] <operator_ns> <index-image-repo> <index-image-tag> [<csv-version> [<install-channel>]]" >&2
+    echo -e "Example:\n\t$0 -d index-test quay.io/rhacs-eng/stackrox-operator-index v3.70.1 v3.70.0" >&2
```

Comment:
Let's make the example complete:
```suggestion
    echo -e "Example:\n\t$0 -d index-test quay.io/rhacs-eng/stackrox-operator-index v3.70.1 v3.70.0 latest" >&2
```